.changeset/honest-jokes-move.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-studio": patch
+"@finos/legend-graph": patch
+---
+
+Properly handle duplicated attributes in `profile`, `enumeration`, `class`, `association`.
```

Comment:
Actually due to the workaround we introduced, I would split this into 2 changesets:

1. `@finos/legend-studio`: Show warning about duplicated attributes in UML editors.
2. `@finos/legend-graph`: Show warning when detecting duplicated attributes in `profile`, `enumeration`, `class`, `association`.